nucypher/cli/commands/ursula.py

```diff
@@ -479,7 +484,16 @@ def config(general_config, config_options, config_file, force, action):
 
     """
     emitter = setup_emitter(general_config, config_options.operator_address)
+
     if not config_file:
+        if action == "migrate":
```

Comment:
Does `migrate` need to be added to the documentation for the `config(...)` function. Only `ip-address` is documented as a sub-command at the moment.

Reply:
Yep - it's a good idea.  We rushed this PR into a merge in order to get multichain deployed on the lynx nodes.  This is related to wider operator-facing documentation.

Reply:
Cool. No worries - can be added later.